singularity-opac/neutrinos/mean_opacity_neutrinos.hpp

```diff
@@ -0,0 +1,212 @@
+// ======================================================================
+// © 2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+// ======================================================================
+
+#ifndef SINGULARITY_OPAC_NEUTRINOS_MEAN_OPACITY_NEUTRINOS_
+#define SINGULARITY_OPAC_NEUTRINOS_MEAN_OPACITY_NEUTRINOS_
+
+#include <cmath>
+
+#include <ports-of-call/portability.hpp>
+#include <singularity-opac/base/radiation_types.hpp>
+#include <singularity-opac/constants/constants.hpp>
+#include <spiner/databox.hpp>
+
+namespace singularity {
+namespace neutrinos {
+
+using pc = PhysicalConstants<CGS>;
+
+#define EPS (10.0 * std::numeric_limits<Real>::min())
```

Comment:
The old `#define EPS (10.0 * std::numeric_limits<Real>::epsilon())` doesn't work for small numbers, e.g. a typical neutrino opacity of `1.e-20 cm^2/g`

Reply:
That seems fine. Is there a reason we did `#define` and not `constexpr Real`? The latter seems better...

Reply:
also should we get rid of the 10?

Reply:
I remember why we have the `#define`... #17 I'm not sure if there's a good way around that (or a good way to have only one definition of that since you'd need an un-`undef`-ed `define` in a header file)

Reply:
oooh. Ok. I guess we can move once we move to C++17.